Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/Core-journeys/sponsored-conversion.cy.js

```diff
@@ -2,17 +2,18 @@
 
 import projectList from "../../pages/projectList";
 import projectTaskList from "../../pages/projectTaskList";
-import projectAssignment from "../../pages/projectAssignment";
+import ProjectAssignment1 from "../../pages/projectAssignment1";
```

Comment:
If you remove the `1` from the import name, and just have the 1 on the filename it means anywhere you are referencing `ProjectAssignment` it won't have a 1 in it :D